docs/griptape-framework/structures/tasks.md

```diff
@@ -98,6 +114,10 @@ If the model supports it, you can also pass image inputs:
 
 ## Tool Task
 
+!!! warning
+
+    `ToolTask` is deprecated and will be removed in a future version. Use [Prompt Task](./tasks.md#prompt-task) with [Reflect On Tool Use](./tasks.md#reflect-on-tool-use) instead.
```

Comment:
Make sure to specify that `reflect-on-tool-use` should be set to False to get the same result as a ToolTask